src/lib/viem-client.ts

```diff
@@ -19,6 +19,15 @@ export const viemEthMainnetClient = createPublicClient({
   transport: http(getConfig().REACT_APP_ETHEREUM_NODE_URL),
 })
 
+export const viemEthMainnetRetryClient = createPublicClient({
+  chain: mainnet,
+  transport: http(getConfig().REACT_APP_ETHEREUM_NODE_URL, {
+    // Retry every 2s, max. 5 times
+    retryCount: 5,
```

Comment:
could we not pop this retry config on the regular client to avoid having two clients? probably not bad to have in general.

Reply:
Very valid point - I'd say we close this as this has been supersed by #5594 - the rationale being failures are usually hollistic and a retry would only spam things more.
Would definitely help in the case of e.g a temporary network issue, so I guess it all comes down to whether or not we're happy with potentially spamming the JSON-RPC endpoint

Reply:
@0xdef1cafe closing this bad boi